### PR TITLE
fix: Update table names in DAG microsoft_store

### PR DESCRIPTION
**Pull request by lucia-vargas-a**
## Description

This PR updates the table names in DAG microsoft_store

## Related Tickets & Documents
* [Bug ](https://bugzilla.mozilla.org/show_bug.cgi?id=1903600)

### CODE REVIEW

File: dags/microsoft_store.py [hunk 1]
```diff
@@ -63,57 +63,26 @@
...
-        owner="mhirose@mozilla.com",
-        email=["mhirose@mozilla.com", "telemetry-alerts@mozilla.com"],
+    table_names = (
+        "app_acquisitions",
+        "app_conversions",
+        "app_installs",
     )
+    for table in table_names:
+        GKEPodOperator(
+            task_id="microsoft_derived__{table}__v1",
```
Comment (BenWu) [inline dags/microsoft_store.py:73]:
```suggestion
            task_id=f"microsoft_derived__{table}__v1",
```

File: dags/microsoft_store.py [hunk 2]
```diff
@@ -63,57 +63,26 @@
...
+            secrets=[
+                microsoft_client_id,
+                microsoft_client_secret,
+                microsoft_tenant_id,
+                microsoft_store_app_list,
+            ],
+            arguments=[
+                "python",
+                "sql/moz-fx-data-shared-prod/microsoft_derived/{table}_v1/query.py",
```
Comment (BenWu) [inline dags/microsoft_store.py:82]:
```suggestion
                f"sql/moz-fx-data-shared-prod/microsoft_derived/{table}_v1/query.py",
```

**Review by Marlene-M-Hirose — APPROVED** [review]

**Review by BenWu — COMMENTED** [review]
It think the pr title `fix:` prefix as well

**Review by BenWu — APPROVED** [review]

**Comment by lucia-vargas-a** [comment]
> It think the pr title needs the `fix:` prefix as well

Thanks!

**Pull request merged**